packages/dnb-eufemia/src/extensions/forms/DataContext/Provider/__tests__/Provider.test.tsx

```diff
@@ -2611,6 +2611,35 @@ describe('DataContext.Provider', () => {
             ).toBeNull()
           })
         })
+
+        it('should override GlobalStatus title when providing one', async () => {
+          jest.spyOn(window, 'scrollTo').mockImplementation()
+          const myTitle = 'Custom title for global status'
+          render(
+            <>
+              <GlobalStatus id="my-status" text={myTitle} />
```

Comment:
Should this be `title`?

```suggestion
              <GlobalStatus id="my-status" title={myTitle} />
```